gf-guide/metrics.md

```diff
@@ -151,22 +151,22 @@ Expected result: vertical metrics should be around 130% of UPM. Anything greater
 A new Latin family has the following qualities:
 
 -   UPM is `1000`
--   `yMax` of tallest `A breve acute` in the familly (black for this example) = `1015`
--   `yMin` of deepest a-z letter (`g` bold in this family) = `210`
+-   `Abreveacute U+1EAE` glyph `yMax` value (taller in the family, Black font for this example) = `1015`
+-   Deepest a-z glyph in the family `yMin` value (`g` bold in this example) = `-210`
 -   Caps height (`H`or `Z` bbox height) = `700`
--   Family's `yMax = 1116` (Black `A breve hookabove` for this family)
--   Family's `yMin = -260` (Black `c cedilla` for this family)
+-   Family's `yMax = 1116` (Black `Abrevehookabove U+1EB2` for this family)
+-   Family's `yMin = -260` (Black `ccedilla U+00E7` for this family)
 
 1.  Set the default values, following the schema above:
 
 ``` code
-typoAscender = 1015 # which matches tallest `A breve acute` in the family and is ≈ [(UPM * 1.3 - CapsHeight) / 2] + CapsHeight
```

Comment:
Why remove the formula?

Reply:
I don't find it helpful to use absolute math when dealing with a family's vertical metrics as it may bring more confusion than clarity. In typography, almost nothing is mathematically calculated, but rather it is about proportions. For instance, once the height of Abreveacute has been established, it can give a sense of how much it's being added to the Caps height. In this case, with a Caps height of 700, it was +315, but according to the math formula, the value should be 300 instead. 

People tend to think that math formulas are more precise, and hence follow them blindly.

If the height of the `Abreveacute` is going way too up, and will force the average VM to be insanely high, e.g. way beyond the suggested 130%, then the stacked marks position or size should be reviewed to fit that percentage.

Reply:
I brought this valuable insight into the document with an extra commit :) PTAL

Reply:
Great! Thank you. I followed it up with another edit + several other editing fixes. PTAL ;)